From a962a83d4002fd053bd392d97ca32ea2a509c398 Mon Sep 17 00:00:00 2001
From: jeebuscrossaint <thediamond270@gmail.com>
Date: Sat, 15 Aug 2026 21:28:37 -0400
Subject: [PATCH 11/29] config: rewrite setlayout keybinds for the 12-layout
 array

Five binds had drifted to layouts[3] and Mod+f pointed at tile instead of
floating, as each layout patch appended without renumbering.
---
 config.def.h | 22 ++++++++++++----------
 1 file changed, 12 insertions(+), 10 deletions(-)

diff --git a/config.def.h b/config.def.h
index a734931..53bbd45 100644
--- a/config.def.h
+++ b/config.def.h
@@ -157,16 +157,18 @@ static const Key keys[] = {
 	{ MODKEY,                    XKB_KEY_Return,      zoom,             {0} },
 	{ MODKEY,                    XKB_KEY_Tab,         view,             {0} },
 	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_c,           killclient,       {0} },
-	{ MODKEY,                    XKB_KEY_t,           setlayout,        {.v = &layouts[0]} },
-	{ MODKEY,                    XKB_KEY_f,           setlayout,        {.v = &layouts[1]} },
-	{ MODKEY,                    XKB_KEY_m,           setlayout,        {.v = &layouts[2]} },
-	{ MODKEY,                    XKB_KEY_r,           setlayout,        {.v = &layouts[3]} },
-	{ MODKEY,                    XKB_KEY_u,           setlayout,        {.v = &layouts[3]} },
-	{ MODKEY,                    XKB_KEY_o,           setlayout,        {.v = &layouts[4]} },
-	{ MODKEY,                    XKB_KEY_c,           setlayout,        {.v = &layouts[3]} },
-	{ MODKEY,                    XKB_KEY_g,           setlayout,        {.v = &layouts[3]} },
-	{ MODKEY,                    XKB_KEY_a,           setlayout,        {.v = &layouts[3]} },
-	{ MODKEY,                    XKB_KEY_s,           setlayout,        {.v = &layouts[3]} },
+	{ MODKEY,                    XKB_KEY_w,           setlayout,        {.v = &layouts[0]} }, /* btrtile        |w|  BSP, you pick the split */
+	{ MODKEY,                    XKB_KEY_t,           setlayout,        {.v = &layouts[1]} }, /* tile           []=  classic master/stack */
+	{ MODKEY,                    XKB_KEY_f,           setlayout,        {.v = &layouts[2]} }, /* floating       ><> */
+	{ MODKEY,                    XKB_KEY_m,           setlayout,        {.v = &layouts[3]} }, /* monocle        [M] */
+	{ MODKEY,                    XKB_KEY_y,           setlayout,        {.v = &layouts[4]} }, /* dwindle        [\]  fibonacci spiral */
+	{ MODKEY,                    XKB_KEY_s,           setlayout,        {.v = &layouts[5]} }, /* bstack         TTT  master top, stack row below */
+	{ MODKEY,                    XKB_KEY_o,           setlayout,        {.v = &layouts[6]} }, /* bstackhoriz    ===   */
+	{ MODKEY,                    XKB_KEY_u,           setlayout,        {.v = &layouts[7]} }, /* centeredmaster |M|  master centred, stack both sides */
+	{ MODKEY,                    XKB_KEY_g,           setlayout,        {.v = &layouts[8]} }, /* gaplessgrid    ### */
+	{ MODKEY,                    XKB_KEY_a,           setlayout,        {.v = &layouts[9]} }, /* deck           [E]  stack becomes a monocle deck */
+	{ MODKEY,                    XKB_KEY_n,           setlayout,        {.v = &layouts[10]} }, /* snail          @|@  spiral master + spiral stack */
+	{ MODKEY,                    XKB_KEY_r,           setlayout,        {.v = &layouts[11]} }, /* mastercol      ||=  master split into columns */
 	{ MODKEY,                    XKB_KEY_space,       setlayout,        {0} },
 	{ MODKEY|WLR_MODIFIER_SHIFT, XKB_KEY_space,       togglefloating,   {0} },
 	{ MODKEY,                    XKB_KEY_e,           togglefullscreen, {0} },
-- 
2.55.0

